sentinel/pkg/checker/peers/app.go

```diff
@@ -0,0 +1,87 @@
+package peers
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"time"
+
+	"bisonai.com/orakl/sentinel/pkg/alert"
+	"bisonai.com/orakl/sentinel/pkg/request"
+	"github.com/rs/zerolog/log"
+)
+
+var peerCount int
+var peerCheckInterval time.Duration
+
+const (
+	DEFAULT_PEER_CHECK_INTERVAL = 10 * time.Second
+	peerCountEndpoint           = "/host/peercount"
+)
+
+type peerCountResponse struct {
+	Count int `json:"Count"`
+}
+
+func setUp() error {
+	var err error
+	peerCheckInterval, err = time.ParseDuration(os.Getenv("PEER_CHECK_INTERVAL"))
+	if err != nil {
+		peerCheckInterval = DEFAULT_PEER_CHECK_INTERVAL
+		log.Error().Err(err).Msgf("Using default peer check interval of %d seconds", DEFAULT_PEER_CHECK_INTERVAL)
+	}
+
+	initialCount, err := checkPeerCounts()
+	if err != nil {
+		return err
+	}
+	peerCount = initialCount
+
+	return nil
+}
+
+func Start() error {
+	err := setUp()
+	if err != nil {
+		return err
+	}
+
+	log.Info().Msg("Starting peer count checker")
+	checkTicker := time.NewTicker(peerCheckInterval)
+	defer checkTicker.Stop()
+
+	for range checkTicker.C {
+		newPeerCount, err := checkPeerCounts()
+		if err != nil {
+			log.Error().Err(err).Msg("Failed to check peer count")
+			continue
+		}
```

Comment:
how about counting how many times it fails checking peer count and if it passes certain threshold (maybe 5-10 times) then send a slack message?